packages/driver/cypress/e2e/dom/visibility.cy.ts

```diff
@@ -367,7 +367,7 @@ describe('src/cypress/dom/visibility', () => {
 `)
 
       this.$elOutOfParentBoundsAbove = add(`\
-<div style='width: 100px; height: 100px; overflow: hidden; position: relative;'>
+<div style='width: 100px; height: 100px; overflow: hidden; position: fixed;'>
```

Comment:
Could you explain this change more fully?

Reply:
This is needed to firefox to behave like other browsers  in this tests. Test check el out of bound and this not changed. And in browser it could be seen as same. But Firefox do not like combination of relative and absolute. Maybe in future it will behave same as other browsers. But for pipeline Firefox version this seems like simple walkaround.

Reply:
Alternatively test could stay not changed and { browser: '!firefox' } added to this test.